item/historical/scripts/DatasetMerger.ipynb

```diff
@@ -346,6 +334,11 @@
     "list_of_paths.append(\"/Users/hlinero/Desktop/iTEM Material/raw dataset/T004/T004_cleaned_PF.csv\")\n",
     "list_of_paths.append(\"/Users/hlinero/Desktop/iTEM Material/raw dataset/T005/T005_cleaned_PF.csv\")\n",
     "list_of_paths.append(\"/Users/hlinero/Desktop/iTEM Material/raw dataset/T006/T006_cleaned_PF.csv\")\n",
+    "list_of_paths.append(\"/Users/hlinero/Desktop/iTEM Material/raw dataset/T007/T007_cleaned_PF.csv\")\n",
```

Comment:
Remove hard-coded paths that are specific to one user's machine.

Reply:
This file is going to change, possibly be erased soon, but I still need it for now. So, we can either ignore it in the gitignore or just wait a bit until I change it.